Jellyfin.Data/Entities/UserData.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Jellyfin.Data.Entities;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
New classes should have the properties documented instead of suppressing warnings